library/ziti.c

```diff
@@ -179,15 +179,8 @@ int ziti_init_opts(ziti_options *options, uv_loop_t *loop, void *init_ctx) {
     uv_timeval64_t start_time;
     uv_gettimeofday(&start_time);
 
-    struct tm *start_tm = gmtime(&start_time.tv_sec);
     char time_str[32];
-#if _WIN32
-    SYSTEMTIME st;
-    GetLocalTime(&st);
-    sprintf(time_str, "%02d-%02d-%04dT%02d:%02d:%02d", st.wDay, st.wMonth, st.wYear, st.wHour, st.wMinute, st.wSecond);
-#else
-    strftime(time_str, sizeof(time_str), "%FT%T", start_tm);
-#endif
+    ziti_fmt_time(&time_str, sizeof(time_str), start_time);
```

Comment:
"&" is technically not the correct way to pass an array as a pointer, because an array is a pointer. If you watch your build you'll probably see a warning on this line:

```
/Users/scarey/repos/github.com/openziti/ziti-sdk-c/library/ziti.c:183:19: warning: incompatible pointer types passing 'char (*)[32]' to parameter of type 'char *'
      [-Wincompatible-pointer-types]
    ziti_fmt_time(&time_str, sizeof(time_str), start_time);
                  ^~~~~~~~~
```

You either want `&time_str[0]` or just plain-old `time_str`.